docs/content/tutorials/best-practices.md

```diff
@@ -130,7 +130,7 @@ namespace MySqlConnector.Examples
         {
             using (var db = new AppDb())
             {
-                await db.OpenAsync();
+                await db.connection.OpenAsync();
                 var cmd = db.Connection.CreateCommand();
```

Comment:
`MySqlCommand` is `IDisposable`; our implementation doesn't do anything special in `Dispose`, but it might be good to show "ideal" usage with a `using` statement.